test/src/test/java/org/corfudb/runtime/view/stream/StreamAddressSpaceTest.java

```diff
@@ -148,7 +148,6 @@ public void testToArray() {
 
     @Test
     public void testEquality() {
-        assertThat(new StreamAddressSpace().equals(null)).isFalse();
```

Comment:
Can you explain why this is being removed? It seems like this was explicitly testing the first condition in the `equals` method of `StreamAddressSpace`.

Reply:
The result of 'new StreamAddressSpace().equals(null)' is always 'false', since when we do `new StreamAddressSpace()` the object is always non-null.

Reply:
We had an offline discussion and decide to put this back, as it's a test for the first condition in the `equals` method of `StreamAddressSpace`.